www/waterfall_view/src/module/main.module.js

```diff
@@ -354,6 +354,9 @@ var WaterfallController = (function() {
          * Draw x axis
          */
         drawXAxis() {
+            if (this.s.show_builders_without_builds.value) {
+                this.builders = this.all_builders;   
```

Comment:
I would rather remove the filtering of this.builders. Did you try that?

Reply:
Updated.